pages/common/redshift.md

```diff
@@ -1,7 +1,7 @@
 # redshift
 
 > Adjust the color temperature of your screen according to your surroundings.
-> More information: <https://jonls.dk/redshift>.
+> More information: <http://jonls.dk/redshift>.
```

Comment:
This is certainly odd.

Reply:
I tweeted the developer: <https://twitter.com/SBRLabs/status/1381266346562965504?s=20>

Reply:
Good job—hopefully they'll get back. Doesn't matter much though ;)

Reply:
Yeah - haven't heard anything yet. This should not block this PR though.